packages/core/src/common/audioAlertPlayer.ts

```diff
@@ -0,0 +1,112 @@
+import { Howl } from 'howler';
+import { leastSevere, mostSevere, defaultAudioSrc } from './constants';
+
+export class AudioAlertPlayer {
```

Comment:
The java in me wants you to make the interface for this player explicit :)

Reply:
It seems like in `IotBlackPearlApp` classes aren't explicit; `eslint` also doesn't let me do that :)

Reply:
How about defining an interface and having this class implement it?